src/metadata.json

```diff
@@ -2,7 +2,7 @@
   "description": "Bring back the transparent top bar when free-floating in GNOME Shell 3.32.\n\nThis basically comes from the feature implementation removed in GNOME Shell 3.32, and I modified the code a bit to make it an extension. Enjoy!",
   "name": "Transparent Top Bar",
   "shell-version": [
-    "45"
+    "45", "46"
```

Comment:
I think we can just drop support for 45 in the next version and simplify the logic - we dropped support for 44 earlier anyway and we can keep doing that.